proto/api_v3/query_service.proto

```diff
@@ -27,6 +27,10 @@ option java_package = "io.jaegertracing.api_v3";
 message GetTraceRequest {
   // Hex encoded 64 or 128 bit trace ID.
   string trace_id = 1;
+  // Optional. The start time to search trace id
+  optional google.protobuf.Timestamp start = 2;
+  // Optional. The end time to search trace id
+  optional google.protobuf.Timestamp end = 3;
```

Comment:
CI fails on the `optional` keyword. IIRC all fields are optional by default, so let's drop the keyword.